File: qcglobal.Core/Domain/department.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace qcglobal.Core.Domain
{
    public class department
    {
        public virtual int id { get; set; }
        public virtual int branchid { get; set; }
        public virtual string departmentcode { get; set; }
        public virtual string departmentname { get; set; }
        public virtual string description { get; set; }
        public virtual int orderidx { get; set; }
        public virtual int parentid { get; set; }
        public virtual bool isparentnode { get; set; }
        public virtual bool isactive { get; set; }
        public virtual int? created_by { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual int? last_updated_by { get; set; }
        public virtual DateTime? last_updated_date { get; set; }
    }
}

File: qcglobal.FEW/ClientApp/src/app/models/department.ts
import { branch } from "./branch";

export interface department {
    id: number;
    branchid: number;
    departmentcode: string;
    departmentname: string;
    description: string;
    order?: number;
    parentid?: number;
    isparentnode?: boolean;
    isactive?: boolean;
    created_by?: number;
    created_date?: Date;
    last_updated_by?: number;
    last_updated_date?: Date;
}
export interface department_edit {
    obj_edit: department;
    list_branch:  { key: string, value: string }[];
}